soc/riscv/riscv-privilege/hi3861/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+# Copyright (c) 2023 Chen Xingyu <hi@xingrz.me>
```

Comment:
This file seems to be empty now? Remove it if it's not needed.

Reply:
Yes, it's empty. But it is required by https://github.com/zephyrproject-rtos/zephyr/blob/c73133d102fb7a5d847bb2dd903e2daaffcd78a9/soc/riscv/riscv-privilege/CMakeLists.txt#L4

Reply:
Maybe I can leave it an no-op `zephyr_sources()`, just like https://github.com/zephyrproject-rtos/zephyr/blob/c73133d102fb7a5d847bb2dd903e2daaffcd78a9/soc/riscv/riscv-privilege/miv/CMakeLists.txt#L3